### PR TITLE
Post: How did we live-stream our LFT of Sept 2021?

### PR DESCRIPTION
**Pull request by pmartin**
Here's a (second) draft for a new blog post, where I write about how we live-streamed our September 2021 LFT.

I don't have a good header/banner image for now :-(
We host a meetup in this auditorium tomorrow, I'll try to snap a couple photos then ^^ (one of the talks has been given during the LFT I'm talking in this post, so the photo will not really be a lie…)

### CODE REVIEW

File: _posts/2021-10-14-live-streaming-lft.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+[^google-live-stream]: [Google Live Stream](https://support.google.com/meet/answer/9308630). We use this solution because Google Meet alone does not support enough people in a call.
+
+We thought we were going to do this in a bit of an ugly way, like when we were all telecommuting: each speaker joins the Google Meet, shares their screen, and speaks into their microphone, often the headset provided by the company. But, still... We decided to make an effort and try to provide a better experience for our colleagues attending LFT remotely!
+
+## So, how we did it, this time
+
+How did we capture and stream slides (or even code and video!), audio, and video of each speaker?
+
+### We start from our amphitheater
```
Comment (taufort) [inline _posts/2021-10-14-live-streaming-lft.md:50]: Attention to the tenses you use. You're using present here but you were using past just above: `how we did it`, `how did we capture`

File: _posts/2021-10-14-live-streaming-lft.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+![An HDMI capture box, close-up view](/images/posts/lft/live-streaming/schema-05-hdmi-capture.jpg)
+
+On the control / broadcast PC, the capture box connected via USB is recognized as a webcam. It is then used as a video input device in OBS.
+
+Hardware: 
+
+ * additional HDMI cable;
+ * capture device + passthrough: [Elgato HD60 S+](https://www.elgato.com/en/game-capture-hd60-s-plus);
+ * usb-c to usb-a cable;
```
Comment (taufort) [inline _posts/2021-10-14-live-streaming-lft.md:118]: USB should always be upper case in all the article IMHO
```suggestion
 * USB-C to USB-A cable;
```

File: _posts/2021-10-14-live-streaming-lft.md (hunk 1 above)
Comment (taufort) [reply _posts/2021-10-14-live-streaming-lft.md:50]: From what I have seen, you're mostly using past to talk about the LFT setup in the following sections :thinking:

File: _posts/2021-10-14-live-streaming-lft.md (hunk 2 above)
Comment (pmartin) [reply _posts/2021-10-14-live-streaming-lft.md:118]: yes.
those lowercase are stuff remaining from my first notes :-/
I'll do a global search/replace

**Pull request merged**